README.md

```diff
@@ -38,8 +38,8 @@ $ sudo apt-get install rethinkdb
 
 #### **Centos 7**
 ```bash
-sudo wget https://dl.bintray.com/rebirthdb/rpm/centos/7/x86_64/rebirthdb.repo \
-    -O /etc/yum.repos.d/rebirthdb.repo
+sudo wget https://dl.bintray.com/rethinkdb/rpm/centos/7/x86_64/rethinkdb.repo \
```

Comment:
We should check these links exist

Reply:
Neither the new link nor the old link exist:

[Old link](https://dl.bintray.com/rebirthdb/rpm/centos/7/x86_64/rebirthdb.repo)
[New link)(https://dl.bintray.com/rethinkdb/rpm/centos/7/x86_64/rethinkdb.repo)